Makefile

```diff
@@ -294,4 +294,20 @@ clean-dist:				  ## Clean up python distribution directories
 	rm -rf dist/ build/
 	rm -rf *.egg-info
 
+
+
+ANTLR_JAR_FILENAME=antlr-4.11.1-complete.jar
+ANTLR_JAR_DOWNLOAD_URL=https://www.antlr.org/download/$(ANTLR_JAR_FILENAME)
+ANTLR_JAR=$(VENV_DIR)/$(ANTLR_JAR_FILENAME)
+ANTLR_GRAMMARS_REL_PATH=localstack/services/stepfunctions/asl/antlr/
+ANTLR_LIB_PATH=$(VENV_DIR)/lib/python*/*/antlr4/
+
+antlr-tool-download: install
+	test -f $(ANTLR_JAR) || wget -O $(ANTLR_JAR) $(ANTLR_JAR_DOWNLOAD_URL)
+
+aslgrammars: antlr-tool-download
+	java -jar $(ANTLR_JAR) -Dlanguage=Python3 $(ANTLR_GRAMMARS_REL_PATH)*Lexer.g4 -o $(ANTLR_LIB_PATH)
+	java -jar $(ANTLR_JAR) -visitor -Dlanguage=Python3 $(ANTLR_GRAMMARS_REL_PATH)*Parser.g4 -lib $(ANTLR_LIB_PATH)$(ANTLR_GRAMMARS_REL_PATH) -o $(ANTLR_LIB_PATH)
```

Comment:
i'm a bit concerned about making this an integral part of the install procedure. is there some way we could make this optional or externalize the building of stepfunctions into a separate folder?
@dominikschubert maybe it makes sense to create a directory for build scripts of various dependencies?